source/what_is_probability.Rmd

```diff
@@ -115,6 +115,104 @@ on developing causality models and is still an exciting, active field of researc
 The rest of this chapter is about the general considerations that are important for principled
 reasoning in the presence of uncertainty.
 
+## Understand your problem
+
+In practice you will asked to provide answers based on data. For example, you may be given data about customer
+behaviour in a large bank and asked to develop a model that will provide the probability of default of the customers in
+the bank. This is an important problem for all financial institutions - if it does not have a good credit risk model,
+it will either loose money by being too conservative in the way it lends money, or loose money bay taking on too much
+of a risk. You can opt for applying a sophisticated model such as a deep neural network but you are almost guaranteed
```

Comment:
Won't they be confused by "deep neural network" here?   Is there a more general way of saying this - such as:

```suggestion
of a risk. You can opt for applying an extremely complex "machine-learning" model with many parameters but you are almost guaranteed
```